/usr/lib/python2.7/dist-packages/scipy/lib/_util.py:35: DeprecationWarning: Module scipy.linalg.blas.fblas is deprecated, use scipy.linalg.blas instead
  DeprecationWarning)
/usr/lib/python2.7/dist-packages/scipy/lib/_util.py:35: DeprecationWarning: Module scipy.linalg.blas.fblas is deprecated, use scipy.linalg.blas instead
  DeprecationWarning)
/usr/lib/python2.7/dist-packages/scipy/lib/_util.py:35: DeprecationWarning: Module scipy.linalg.blas.fblas is deprecated, use scipy.linalg.blas instead
  DeprecationWarning)
/usr/lib/python2.7/dist-packages/scipy/lib/_util.py:35: DeprecationWarning: Module scipy.linalg.blas.fblas is deprecated, use scipy.linalg.blas instead
  DeprecationWarning)
/usr/lib/python2.7/dist-packages/scipy/lib/_util.py:35: DeprecationWarning: Module scipy.linalg.blas.fblas is deprecated, use scipy.linalg.blas instead
  DeprecationWarning)
/usr/lib/python2.7/dist-packages/scipy/lib/_util.py:35: DeprecationWarning: Module scipy.linalg.blas.fblas is deprecated, use scipy.linalg.blas instead
  DeprecationWarning)
/usr/lib/python2.7/dist-packages/scipy/lib/_util.py:35: DeprecationWarning: Module scipy.linalg.blas.fblas is deprecated, use scipy.linalg.blas instead
  DeprecationWarning)
/usr/lib/python2.7/dist-packages/scipy/lib/_util.py:35: DeprecationWarning: Module scipy.linalg.blas.fblas is deprecated, use scipy.linalg.blas instead
  DeprecationWarning)
Using gpu device 0: GRID K520
0.693347200563
loading data
Training for Protists
epoch 0/1, minibatch 1437/1437, validation error 88.950276 %
Traceback (most recent call last):
  File "main.py", line 46, in <module>
    print 'Score for ' + name + ': ' + str(online_score(predictions, test_y))
  File "/mnt/planktonChallenge/code/score.py", line 27, in online_score
    return logloss(targets, predictions)
  File "/mnt/planktonChallenge/code/score.py", line 12, in logloss
    return -T.mean(T.log(pred)[T.arange(act.shape[0]), act]).eval()
  File "/usr/local/lib/python2.7/dist-packages/theano/tensor/var.py", line 388, in __getitem__
    return theano.tensor.subtensor.advanced_subtensor(self, *args)
  File "/usr/local/lib/python2.7/dist-packages/theano/gof/op.py", line 490, in __call__
    node = self.make_node(*inputs, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/theano/tensor/subtensor.py", line 1871, in make_node
    index = tuple(map(as_index_variable, index))
  File "/usr/local/lib/python2.7/dist-packages/theano/tensor/subtensor.py", line 1815, in as_index_variable
    raise TypeError('index must be integers')
TypeError: index must be integers
